FF1Lib/Flags.cs

```diff
@@ -194,14 +195,19 @@ public class Flags : IIncentiveFlags, IMapEditFlags
 		[FlagString(Character = 19, Multiplier = 0.1)]
 		public double ExpMultiplier { get; set; }
 		[FlagString(Character = 20, Multiplier = 10)]
-		public int ExpBonus { get; set; }
-		[FlagString(Character = 21, Multiplier = 1)]
+		public double ExpBonus { get; set; }
+		[FlagString(Character = 21)]
 		public int ForcedPartyMembers { get; set; }
-		
+
+		[FlagString(Character = FUN_PERCENT, FlagBit = 1)]

```

Comment:
We don't want the fun% stuff in the flags string because they don't affect the game and thus different people can set them differently and still have the same experience.